diff --git a/node_modules/react-native-auth0/android/.project b/node_modules/react-native-auth0/android/.project
new file mode 100644
index 0000000..1f69f8d
--- /dev/null
+++ b/node_modules/react-native-auth0/android/.project
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>react-native-auth0</name>
+	<comment>Project react-native-auth0 created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1780040033394</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/react-native-auth0/ios/A0Auth0.h b/node_modules/react-native-auth0/ios/A0Auth0.h
index e4bec85..70a5bfd 100644
--- a/node_modules/react-native-auth0/ios/A0Auth0.h
+++ b/node_modules/react-native-auth0/ios/A0Auth0.h
@@ -1,4 +1,4 @@
-#ifdef RCT_NEW_ARCH_ENABLED
+#if defined(RCT_NEW_ARCH_ENABLED) && defined(__cplusplus)
 #import "RNAuth0Spec.h"
 
 @interface A0Auth0 : NSObject <NativeA0Auth0Spec>
